### PR TITLE
add road update info for sept2021

### PR DESCRIPTION
**Pull request by gregbunce**
also: 

- update ng911 and network dataset dates.  
- add link to transportation page. 
- update recreation index page trails entry to add pathways
- add trails entry to transportation index page

### CODE REVIEW

**Comment by gregbunce** [comment]
it looks good to me.  let me know if you want any changes.  feel free to merge it if you are happy with it.  thanks!

File: _posts/2021-09-03-utah-sgid-statewide-roads-data-layer-updates-09032021.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+#### County Updates
+
+New roads were added and road names and address ranges were updated for the following counties:
+
+- **Cache County:** Obtained roads data on 08/17/2021. Previous update was on 06/04/2021.
+- **Davis County:** Obtained roads data on 08/19/2021. Previous update was on 08/06/2021.
+- **Utah County:** Obtained roads data on 08/17/2021. Previous update was on 08/06/2021.
+- **Washington County:** Obtained roads data on 08/18/2021. Previous update was on 08/06/2021.
+
+Note: Data was requeted but not received from Juab County, Kane County, Tooele County, and Weber County. Data was received from Morgan County and Salt Lake County but due to internal processing issues it was not processed for this month.
```
Comment (steveoh) [inline _posts/2021-09-03-utah-sgid-statewide-roads-data-layer-updates-09032021.md:28]: Spelling

**Review by steveoh — COMMENTED** [review]
One spelling fix and your gtg

File: _posts/2021-09-03-utah-sgid-statewide-roads-data-layer-updates-09032021.md (hunk 1 above)
Comment (gregbunce) [reply _posts/2021-09-03-utah-sgid-statewide-roads-data-layer-updates-09032021.md:28]: oops, good catch!  thx

**Review by steveoh — APPROVED** [review]

**Pull request merged**